### PR TITLE
add fix and UT

### LINKED ISSUE #6: nil pointer panic
Hi! Thank you for this handy wrapper. But in the process of implementation, I found one simple bug.

If I try to calculate something in the following way:
```
new(int256.Int).Mul(a, b)
```
Then `z.abs` will be nil and we will get a panic. Same is true for other operations.

Fix is quite simple - add a check for nil and if it's true, initialize `abs` before operations. And update unit tests.

p.s. I think this wrapper will earn a lot more stars if you fix it :)

### PR DESCRIPTION
**Pull request by rajkumarGosavi**
closes #6

### CODE REVIEW

**Comment by linhbkhn95** [comment]
Great job. Thank @rajkumarGosavi.
I will review it.

File: int256.go [hunk 1]
```diff
@@ -337,3 +363,12 @@ func (z *Int) And(x, y *Int) *Int {
 	z = MustFromBig(big)
 	return z
 }
+
+// setZero sets default `0` value if `z` is nil
+func (z *Int) setZero() *Int {
```
Comment (linhbkhn95) [inline int256.go]: the naming is confusing a bit. Moreover, function `set` but returning int

Comment (rajkumarGosavi) [reply int256.go]: Makes sense, we can directly do `z.abs = new(uint256.Int)` if that's alright i.e. we directly update the receivers `abs` field value.

Function name can then be `initiateAbs()` `initAbs()`, please suggest if we can we give it a better name.

Thanks !!

Comment (linhbkhn95) [reply int256.go]: two names above is okay for me

Comment (rajkumarGosavi) [reply int256.go]: I have made the respective changes, please review.

**Review by linhbkhn95 — APPROVED** [review]

File: int256.go (hunk 1 above)
Comment (rajkumarGosavi) [reply int256.go]: Hi @linhbkhn95 I have made the changes as suggested

**Pull request merged**